build.gradle

```diff
@@ -1,32 +1,27 @@
 plugins {
-	id 'fabric-loom' version '0.2.7-SNAPSHOT'
+	id 'fabric-loom' version '0.8-SNAPSHOT'
 	id 'maven-publish'
 }
 
-sourceCompatibility = JavaVersion.VERSION_1_8
-targetCompatibility = JavaVersion.VERSION_1_8
+sourceCompatibility = JavaVersion.VERSION_16
+targetCompatibility = JavaVersion.VERSION_16
```

Comment:
Is it possible to keep these on Java 1.8 so we can remain compatible with 1.16?

Reply:
I don't think so, because updating to loom 0.8-SNAPSHOT breaks compatibility with Java <16, and is the recommended version to use for 1.17 (I believe it is also required, but I am not sure).

Reply:
I think it should still work to compile with Java 16, just targeting Java 8

Reply:
I believe that loom 0.8 makes the mod not work on java <16, I read that here: https://github.com/astei/lazydfu/commit/87b88caae59fd66487460223f03efa0a3ff823a0#commitcomment-51761705